packages/veritone-widgets/src/widgets/MediaDetails/index.js

```diff
@@ -622,18 +618,37 @@ class MediaDetailsWidget extends React.Component {
     if (!this.isEditableEngineResults()) {
       return false;
     }
-    if (get(this.props.selectedEngineCategory, 'categoryType') === 'face') {
-      const selectedEngine = find(this.props.selectedEngineCategory.engines, {
-        id: this.props.selectedEngineId
-      });
-      return (
-        !selectedEngine.hasUserEdits || this.props.isDisplayingUserEditedOutput
-      );
+    const selectedCategoryType = get(this.props.selectedEngineCategory, 'categoryType');
+    const selectedEngine = find(this.props.selectedEngineCategory.engines, { id: this.props.selectedEngineId });
+    if (selectedCategoryType === 'face' && get(selectedEngine, 'hasUserEdits') && !this.props.isDisplayingUserEditedOutput) {
```

Comment:
maybe change this to something like this?

const editableCategoryTypes = ['face', 'transcript'];
// includes is from lodash
if (includes(editableCategoryTypes, selectedCategoryType) && get(selectedEngine, 'hasUserEdits') && !this.props.isDisplayingUserEditedOutput) {
    return false;
}

Then if we add a new editable category type just add it to the array.